src/vs/workbench/contrib/remote/browser/remoteIndicator.ts

```diff
@@ -301,7 +301,11 @@ export class RemoteStatusIndicator extends Disposable implements IWorkbenchContr
 			const workspaceLabel = this.getWorkspaceLabel();
 			if (workspaceLabel) {
 				const toolTip: IMarkdownString = {
-					value: nls.localize('workspace.tooltip', "Virtual workspace on {0}\n\n[Some features](command:{1}) are not available for resources located on a virtual file system.", workspaceLabel, LIST_WORKSPACE_UNSUPPORTED_EXTENSIONS_COMMAND_ID),
+					value: nls.localize(
```

Comment:
This is the only one I didn't know how to test... the others I manually tested still worked in English.